man/adoc/bpftrace.adoc

```diff
@@ -1950,7 +1950,7 @@ For string arguments in an action block use the `str()` call to retrieve the val
 | uint32
 | 4.2
 | get_current_pid_tgid
```

Comment:
We should probably update this too for both
```suggestion
| get_current_pid_tgid / bpf_get_ns_current_pid_tgid
```

Reply:
Done!